Common/Utils/include/CommonUtils/RootSerializableKeyValueStore.h

```diff
@@ -0,0 +1,175 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_ROOTSERKEYVALUESTORE_H
+#define ALICEO2_ROOTSERKEYVALUESTORE_H
+
+#include <map>
+#include <string>
+#include <TClass.h>
+#include <Rtypes.h>
+#include <typeinfo>
+#include <typeindex>
+#include <iostream>
```

Comment:
I would avoid exposing iostream in an header, especially if it's needed to be included a bit everywhere.

Reply:
Yes of course. Will rework it.